README.md

```diff
@@ -233,32 +250,35 @@ $ tree my-component
 First, install the cli and then use `manifest-gen` to do this.
 
 ``` shell
-$ cd modality-probe-cli
+$ cd ../modality-probe-cli
 $ cargo install --path .
-$ cd ../
+$ cd ../modality-probe-capi
```

Comment:
This should just be back to the root path, or we should skip it. You don't need to be in the c api dir to run the command, and you're pointing it at a different directory anyway, so this is confusing

Reply:
:+1: This is inherited from the original `modality-probe-capi` README.md; I couldn't tell you why. Changing to:
`$ cd ../`